trio/_core/_multierror.py

```diff
@@ -6,6 +6,9 @@
 from contextlib import contextmanager
 
 import attr
+import logging
```

Comment:
I don't think this import is used at all anymore. I should remove it.

Reply:
👍